korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt

```diff
@@ -1679,22 +1679,22 @@ fun <T : View> T.centerBetween(x1: Double, y1: Double, x2: Double, y2: Double):
  * Chainable method returning this that sets [View.x] so that
  * [this] View is centered on the [other] View horizontally
  */
-fun <T : View> T.centerXOn(other: View): T = this.centerXBetween(other.x, other.x + other.width)
```

Comment:
This only was guaranteed to work if  `this` and `other` have the same parent (and thus the same parent.globalTransform)

Some people did:

```
view.centerXOnStage()
```

And since stage.x might not be 0, specilaly on JS where the aspect ratio might be different, this caused the centered text to appear to the right since `(stage.x + stage.width) / 2` is not the center